components/stripe/actions/create-usage-record/create-usage-record.js

```diff
@@ -23,20 +23,31 @@ module.exports = {
       "propDefinition": [
         stripe,
         "price",
+        () => ({
+          // Only `recurring` prices can be used to filter subscriptions
+          type: "recurring",
```

Comment:
I would suggest setting this kind of fixed values in a `constants.js` file and references from it